usaspending_api/settings.py

```diff
@@ -56,6 +56,7 @@
 CFDA_BUCKET_NAME = ""
 CFDA_REGION = ""
 CFDA_FILE_PATH = ""
+ROSETTA_DICT_S3_PATH = "da-public-files/user_reference_docs/DATA Transparency Crosswalk.xlsx"
```

Comment:
Should this be set by OPs? I guess since it's public it's fine to include the path.

Reply:
I did this under OPS direction